WMF Framework/WMFLocalization.m

```diff
@@ -0,0 +1,20 @@
+#import "WMFLocalization.h"
+
+static NSString *const WMFLocalizationBundleIdentifier = @"org.wikimedia.WMF";
+
+NSString *WMFLocalizedStringWithDefaultValue(NSString *key, NSURL *url, NSBundle *bundle, NSString *value, NSString *comment) {
```

Comment:
New function for requesting **site specific** localizations. Since this matches the signature of the built in `NSLocalizedStringWithDefaultValue` (with table name replaced with url) we can use the default strings generation tools.